pytest.ini

```diff
@@ -1,5 +1,4 @@
 [pytest]
-addopts = -m "dev and not fluent_231"
```

Comment:
revert the changes from Mainak as this may create confusion when we want to run the whole set of tests.